contrib/opencensus-ext-azure/opencensus/ext/azure/common/transport.py

```diff
@@ -209,6 +217,25 @@ def _transmit(self, envelopes):
                 with _requests_lock:
                     _requests_map['retry'] = _requests_map.get('retry', 0) + 1  # noqa: E501
             return self.options.minimum_retry_interval
+        if response.status_code in (307, 308):
```

Comment:
For 307 case, Cache-Control header would be returned as well specifying how much time the location URL need to be cached

Reply:
Will probably leave this for another PR.